flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/UserDefinedFunction.scala

```diff
@@ -24,4 +24,19 @@ package org.apache.flink.table.functions
   * User-defined functions must have a default constructor and must be instantiable during runtime.
   */
 trait UserDefinedFunction {
```

Comment:
I tried your code and my code again and it works good. Do not know why not work before...

Thank you for your explain.